build.sbt

```diff
@@ -2,28 +2,27 @@ import Defaults._
 
 lazy val root = (project in file("."))
   .settings(commonSettings: _*)
-  .enablePlugins(CrossPerProjectPlugin)
   .aggregate(NGHelpers, NGPlugin)
 
 lazy val NGHelpers = (project in file("helpers"))
   .settings(commonSettings: _*)
   .settings(
     name := "play-testng-helpers",
-    scalaVersion := "2.11.11",
+    scalaVersion := "2.12.4",
     crossScalaVersions := Seq("2.11.11", "2.12.4"),
     libraryDependencies ++= Seq(
-      "org.testng" % "testng" % "6.8.8", // % "provided"
-      "com.typesafe.play" %% "play-test" % "2.5.18", //% "provided"
-      "com.typesafe.play" %% "play-java" % "2.5.18" //% "provided"
+      "org.testng" % "testng" % "6.11", // % "provided"
+      "com.typesafe.play" %% "play-test" % "2.6.7", //% "provided"
+      "com.typesafe.play" %% "play-java" % "2.6.7" //% "provided"
```

Comment:
We need to discuss about this. Since play 2.6.6, it starts to use sbt 1.0. This will introduce more work for us to migrate. I suggest to use Play 2.6.5 for now in order to keep using sbt 0.13.

Reply:
Does the version of SBT that Play itself uses matter? Their announcement for 2.6.7 didn't say that apps need to move to 1.0

Reply:
This shouldn't affect the Play app's SBT version. But any SBT plugin library, which depends on Play's sbt-plugin, would need to move to 1.0.